### ISSUE TITLE
/var/log partition too small

### ISSUE THREAD

**Issue by FuhuXia** [GSA/data.gov#123]
/var/log keeps getting full for some instances such as solr and catalog web instances. By default the partition is 2G. It can get filled up within hours on solr, and days on catalog web, not enough room even for log rotation. We can re-partition it in ami, but for immediate remedy we should attach 30-50GB EBS volumn to all solr and catalog web instances.

**Comment by dano-reisys** [comment]
@neilhunt1 is the latest code for the hardening playbooks @ `https://github.com/GSA/catalog-deploy/tree/master/ansible/roles/security/hardening` ?

**Comment by neilhunt1** [comment]
Dan, [this ](https://github.com/GSA/catalog-deploy/blob/master/ansible/roles/security/hardening/tasks/main.yml)is the main playbook that is run, it fawns off to the sub playbooks mentioned therein.

**Comment by dano-reisys** [comment]
**hardened ubuntu:**
`ubuntu@ip-10-0-0-42:~$ lsblk`
`NAME    MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT`
`xvda    202:0    0     8G  0 disk`
`└─xvda1 202:1    0     8G  0 part /`
`xvdb    202:16   0    30G  0 disk`
`├─xvdb1 202:17   0     8G  0 part`
`├─xvdb2 202:18   0     2G  0 part /var`
`├─xvdb3 202:19   0     2G  0 part /var/log`
`├─xvdb4 202:20   0     4G  0 part /home`
`├─xvdb5 202:21   0   1.2G  0 part /tmp`
`└─xvdb6 202:22   0   1.4G  0 part /var/log/audit`

@neilhunt1 did you partition xvdb ebs-volume manually after running the hardening playbook?

**regular ubuntu:** 
`root@ip-10-0-0-55:/home/ubuntu# lsblk`
`NAME    MAJ:MIN RM SIZE RO TYPE MOUNTPOINT`
`xvda    202:0    0   8G  0 disk`
`└─xvda1 202:1    0   8G  0 part /`
`xvdb    202:16   0  30G  0 disk`

**Comment by neilhunt1** [comment]
No, I did it before running the hardening playbook. When I created the non-hardened AMI I had to do so with an external volume (xvdb) attached already with /etc/fstab pointing to those partitions. Then, I ran the hardening playbook and created a second AMI (the "hardened" one we've been using). If you don't, hardening process will produce negative findings because /var, /var/log, etc. will be on the same partition.
